src/app/home/countdown/countdown.component.scss

```diff
@@ -0,0 +1,64 @@
+#countdown-title {
+  color: #64b5f6;
+}
+
+.card-content {
+  background: #64b5f6;
+}
+
+.title-font {
+  font-size: 20px;
```

Comment:
We are trying to avoid defining font sizes to have an standard all over the page. Use this only if it's really mandatory.